src/SDK/SmartDetectorsShared/Trace/ConsoleTracer.cs

```diff
@@ -8,6 +8,7 @@ namespace Microsoft.Azure.Monitoring.SmartDetectors.Trace
 {
     using System;
     using System.Collections.Generic;
+    using Microsoft.Azure.Monitoring.SmartDetectors.Extensions;
```

Comment:
>Microsoft.Azure.Monitoring.SmartDetectors.Extensions [](start = 10, length = 52)

why?

Reply:
it was as a result of 100 refactoring I did in this PR.. removed all unnecessary ones

---
In reply to: [257936977](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/48#discussion_r257936977) [](ancestors = 257936977)